src/libraries/Microsoft.PowerFx.Interpreter/ReflectionFunction.cs

```diff
@@ -153,4 +155,10 @@ public FormulaValue Invoke(FormulaValue[] args)
             return (FormulaValue)result;
         }
     }
+
+    // A function capable of async invokes. 
+    internal interface IAsyncTexlFunction
```

Comment:
>IAsyncTexlFunction [](http://example.com/codeflow?start=23&length=18)

Currently, just tests implement this. 
Can followup on the proper extension story in a future PR.